dashboard/test/lib/services/script_seed_test.rb

```diff
@@ -628,30 +630,32 @@ def create_script_tree(
 
       script.lesson_groups.each_with_index do |lg, m|
         num_lessons_per_group.times do |n|
-          name = "#{name_prefix}-lg-#{m + 1}-lesson-#{n + 1}"
+          name = "#{name_prefix}-lg-#{m + 1}-l-#{n + 1}"
```

Comment:
I had to make this shorter because the index on the key field in ActivitySection only uses the first so many characters, and this field is included in the key.